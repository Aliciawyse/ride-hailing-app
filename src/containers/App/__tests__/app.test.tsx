// One way to mock component
jest.mock('../../../components/Header');

// Another way to mock component
jest.mock('../../../components/Footer', () => {
  return {
    Footer: () => {
      const React = require('react');
      return (
        <div />
      )
    }
  }
})

import * as React from 'react';

import { observable } from 'mobx';
import { App } from '..';
import * as renderer from 'react-test-renderer';

describe('AppContainer', () => {
  let store;
  beforeEach(() => {

    store = observable({
      app: [],
      i18n: {
        t: () => 'some string'
      }
    });
  });

  test('Component is rendered properly', () => {
    const component = renderer.create(
      <App {...store}/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
})