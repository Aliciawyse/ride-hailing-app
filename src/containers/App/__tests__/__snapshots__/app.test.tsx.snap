// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer Component is rendered properly 1`] = `
<div
  className="normal"
>
  <p />
  <div />
</div>
`;
